CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## 0.6.0
+## 0.7.0
+- **Breaking Change** Pack bundle format is changed to IIFE. Previously compiled bundles will still be supported for a short period of time.
```

Comment:
What change would we make that would break previously compiled bundles? Can we support them forever, it's just that they'll continue to have broken source maps?

Otherwise we have to update every first-party pack again, and ask all beta users to update their packs.

Reply:
we can probably support them forever. maybe sometime we'll be running on pack bundles with sdk <0.7.0. We can remove the support then. 

with that begin said, maybe I'll remove the "breaking change" tag here to avoid confusion.